File: python/td/td2-structures-conditionnelles/additional/exercice1.py
# Programme principal
# variables
#   entrée
#       entier année
# début
#   écrire("Entrez votre année")
#   lire (année)
#   si année % 4 != 0
#       écrire("L'année {année} n'est pas bissextile !")
#   sinon
#       si année % 100 == 0 ET année % 400 != 0
#           écrire("L'année {année} n'est pas bissextile !")
#       sinon
#           écrire("L'année {année} est bissextile !")
#       fin si
#   fin si
# fin
print("Entrez votre année")
annee = int(input())
if annee % 4 != 0:
    print(f"L'année {annee} n'est pas bissextile")
else:
    if (annee % 100 == 0) & (annee % 400 != 0):
        print(f"L'année {annee} n'est pas bissextile")
    else:
        print(f"L'année {annee} est bissextile")

File: cpp/exercices/exercice-1.cpp
#include <iostream>
using namespace std;

int main() {
    int j,m,a;
    cout << "Entrez votre jour de naissance (01 - 31): ";
    cin >> j;
    cout << "Entrez votre mois de naissance (01 - 12): ";
    cin >> m;
    cout << "Entrez votre année de naissance (1900 - 2024): ";
    cin >> a;
    printf("Votre date de naissance est : %02d/%02d/%d\n", j, m, a);
    return 0;
}
